scripts/install_dependencies_apt.py

```
@@ -9,43 +9,54 @@
+# end-of-run failure summary. Keeps the root cause near the end of the log,
+# which is where first_time_install.sh's error handler tails from.
+ERROR_TAIL_LINES = 15
+
+
+def _run(cmd):
+    """Run a command, capturing combined stdout/stderr.
+
+    Returns (success, output) instead of raising, so callers can report
+    *why* a command failed rather than just that it failed.
+    """
+    result = subprocess.run(cmd, capture_output=True, text=True)
```

Ruff (0.15.15)

[error] `subprocess` call: check for execution of untrusted input

(S603)